docs/common-coreboot-docs/dumping_logs.md

```diff
@@ -7,20 +7,27 @@ a running system.
 
 ## System information
 
-One may use [Dasharo Tools Suite HCL report](../../dasharo-tools-suite/documentation#hcl-report)
+One may use [Dasharo Tools Suite HCL report](../dasharo-tools-suite/documentation#hcl-report)
 or [fwdump-docker image](https://github.com/3mdeb/fwdump-docker) to
 gather all the hardware configuration information from a running system. The
 usage of the tools should result in an archive containing various logs from the
 running system. Submit them via email to [contact@dasharo.com](mailto:contact@dasharo.com)
 or use [Dasharo pastebin](https://paste.dasharo.com/).
 
-## coreboot logs
+## `cbmem` utility
 
 When already migrated to Dasharo, it is possible to retrieve firmware logs from
 coreboot on a running system. A utility called `cbmem` can be used for that
 purpose. By obtaining the logs Dasharo team will be able to locate any issues
 with the firmware. This method requires Secure Boot to be disabled.
 
+Options to get `cbmem` utility:
+
+1. Download precompiled utility from [3mdeb cloud](https://cloud.3mdeb.com/index.php/s/zTqkJQdNtJDo5Nd/download)
```

Comment:
Under this link there's not only `cbmem` but also `flashrom` and binary for one of Dasharo products with wrong-signed keys. In my opinion we should change it in order not to cause any confusion.

Reply:
@sulewskiprzemyslaw that's the link you use in your test procedures throughout the whole docs repo. I think we should merge it as it is and then fix all occurrences of the cloud share with a separate share for cbmem only.

Reply:
@miczyg1 okay, I merge this PR and fix in another PR inconsistency with downloading `cbmem`